src/alchemlyb/tests/test_visualisation.py

```diff
@@ -34,7 +35,9 @@ def test_plot_mbar_omatrix():
 def test_plot_ti_dhdl():
     '''Just test if the plot runs'''
     bz = load_benzene().data
-    dHdl_coul = pd.concat([extract_dHdl(xvg, T=300) for xvg in bz['Coulomb']])
+    dHdl_coul = alchemlyb.concat([extract_dHdl(xvg, T=300) for xvg in bz['Coulomb']])
+    dHdl_coul.attrs = extract_dHdl(load_benzene().data['Coulomb'][0],
+                                   T=300).attrs
```

Comment:
Why is the explicit attr assignment needed here? I though `extract_dHdl` and `alchemlyb.concat()` were automatically handling this case??

Reply:
Sorry, I initially did this and then I felt it might be more useful to have alchemlyb.concat() and then I change pd.concat to alchemlyb.concat without removing this.